backend/src/event/schemas/event.schema.ts

```diff
@@ -1,6 +1,22 @@
 import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
 import { Document, Types } from 'mongoose';
 
+enum EventCategory {
+  SORTIE_BAR = 'Sortie bar',
```

Comment:
you should save them in english in your ENUM, with a keyword or anything else, no need to have it correctly formatted backend wise IMO.

regarding the translation that could be done in the Front-End only, he can easily map the key with a new value, while in your case if you want to update one later, that will not work

Also: **English variable names** 👀